Advisor docs (#1347)
scylladb/scylla-monitoring: docs/source/advisor/cqlCLAny.rst

Some queries use Consistency Level: ANY
--------------------------------------

Scylla stores your data in ReplicationFactor of nodes (replicas). A write query Consistency Level determines how many replicas need to acknowledge the write before a reply is returned.

> Using consistency level ANY allows a write query to be acknowledged after storing the information on the coordinator, a non replica (as hints), meaning the data is not yet persistent on disk while acknowledge is sent back to the application. Use CL ANY with care. 

Reviewer: ```suggestion
Using consistency level ANY allows a write query to be acknowledged after storing the information on the coordinator, a non-replica (as hints), meaning the data will be acknowledged when in fact it is not yet persistent on disk. Use Consistency Level ANY with care. 
```
